### ISSUE TITLE
Constellation paper 3.0.6

### ISSUE THREAD

**Issue by Smoketrail** [Darkosto/SevTech-Ages#1729]
## Issue / Bug

Using the Necronomicon ritual to craft a constellation paper gives me a parchment paper back and consumes the 4 aquamarine, 2 gold and 2 crystals.
## Expected Behavior

Expected result: a constellation paper
Result: parchment paper.

## Steps to Reproduce (for bugs)

1. Setup Ritual according to JEI.
2. Use Necronomicon book fully charged, shift right click on center ritual block.
3. Ritual is triggered
4. Results in parchment paper.

## Context

Trying to get Constellation paper to progress in age 2

## Client Information

* Modpack Version: 3.0.6
* Java Version: 1.8.0.131 (64bit)
* Launcher Used: Twitch

* Memory Allocated: 8.7GB

* Server/LAN/Single Player: Server
* Resourcepack Enabled?: No
* Optifine Installed?: Yes

## Server Information
* Java Version: unsure
* Operating System: unsure
* Hoster/Hosting Soultion: ScalaCube
* Sponge (Non-Vanilla Forge) Server?: no

* Additional Content Installed?: none

**Comment by artdude543** [comment]
You need to wait for the entire process to complete before removing items. Sometimes this could be slower due to tick times or network lag.

But this is not a pack issue.

**Comment by RandomLibrarian** [comment]
This is happening to me as well for the first time in a few playthroughs. I waited extra time 3 times in a row.... still parchment paper.

**Comment by Smoketrail** [comment]
completly forgot i made this post. Try it at night, that helped for me

**Comment by Ezelberoth** [comment]
I had this happen once, the only difference between getting the parchment back, and getting the constellation paper, was that when it only gave me parchment back, I had put my necro book back on the energy pedestal before the ritual was complete. I managed to get 5 constellation papers in a row by keeping the necro book in my hand (in the active slot on bar) until the entire ritual was complete, and lightning had struck the middle pedestal (the one holding the parchment), and I saw the constellation paper (identified by the spots on the paper). I am using version 3.0.6 of the pack.

Also, I was able to do this ritual during the day, and during the night.
